### ISSUE TITLE
Changing a symlink into a file in git causes ClearCase checkin to fail

### ISSUE THREAD

**Issue by chrahunt** [charleso/git-cc#71]
This was located along with #70. I replaced a symlink with a file of the same name and committed. When I tried to do `gitcc checkin` I received `KeyError: u'T'` from [this](https://github.com/charleso/git-cc/blob/master/git_cc/checkin.py#L80) line. Per the [`git-diff`](https://git-scm.com/docs/git-diff) docs (under `--diff-filter`) `T` is returned for `type` when the file in question has its type changed between symlink, regular file, and submodule. It looks like the dict [here](https://github.com/charleso/git-cc/blob/master/git_cc/checkin.py#L62) is lacking a suitable handler.

**Comment by charleso** [comment]
@chrahunt Yeah never worried about symlinks. If you haven't already discovered, I don't use/touch git-cc any more. If you manage to add support then I'm happy to accept a pull request, but otherwise I'm afraid there isn't much that I can do. Best of luck.
